client/jetpack-connect/authorize.js

```diff
@@ -276,6 +278,9 @@ export class JetpackAuthorize extends Component {
 				this.isSso()
 			);
 			this.externalRedirectOnce( redirectAfterAuth );
+		} else if ( this.isFromJetpackBackupPlugin() && ! siteHasJetpackBackupProduct ) {
+			debug( `Redirecting directly to cart with `, PRODUCT_JETPACK_BACKUP_T1_YEARLY, ' in cart.' );
```

Comment:
[Strictly optional, non-blocking]

Since "`"-style strings have variable interpolation built in, we could probably do the following:

```suggestion
			debug( `Redirecting directly to cart with ${PRODUCT_JETPACK_BACKUP_T1_YEARLY} in cart.` );
```

Reply:
I thought the linter would take out the `'s for sure 🤷. Strings w/ variable interpolation is best IMO.